src/montagu_deploy/montagu_constellation.py

```diff
@@ -195,3 +250,12 @@ def proxy_configure(container, cfg):
         docker_util.string_into_container(cfg.ssl_certificate, container, join(ssl_path, "certificate.pem"))
         docker_util.string_into_container(cfg.ssl_key, container, join(ssl_path, "ssl_key.pem"))
         docker_util.string_into_container(cfg.dhparam, container, join(ssl_path, "dhparam.pem"))
+
+
+def exec_safely(container, args, **kwargs):
```

Comment:
ideally this should move into `constellation` https://github.com/reside-ic/constellation/blob/master/constellation/docker_util.py#L28